src/constants.ts

```diff
@@ -11,6 +11,14 @@ export const RPC_HTTP_PORT = 9944;
 // The port substrate listens for p2p connections on
 export const P2P_PORT = 30333;
 
+// Paths for downloading the binaries for zombienet setup
+export const DEFAULT_COMMAND_URL =
+  "https://github.com/paritytech/polkadot/releases/download/v0.9.27/polkadot";
```

Comment:
Hi @wirednkod, I think we need to add the logic to download the latest release so we don't need to update this path every new release. For example `v0.9.28` was release two days ago.

Reply:
Could not agree more. This was more of a draft version, to see if we are on correct path